src/js/components/DataTable/__tests__/DataTable-test.tsx

```diff
@@ -552,7 +552,7 @@ describe('DataTable', () => {
             }
             return (
               <Box>
-                {row.a} : {row.b}{' '}
+                {row.a} :{row.b}
```

Comment:
I don't think this file should be changed by this PR, looks unrelated/potentially a local linting change